### DISCUSSION TITLE
Bug: When screen rotation is 270, touch simulated is inversed

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#14] [Genymotion Desktop] [answered]
I'm using Genymotion on Ubuntu 20.04. The virtual device is a custom tablet of resolution 1920x1080. If I run
```
rotation setangle 270
```
in the Genymotion Shell, every simulated touch is inversed from what is intended by the user. In other words, whenever I click at position $(x,y)$, the simulated touch is at position $(w-x,h-y)$, where $w$ and $h$ are width and height of the virtual device's screen.
If this is intended, please tell me the reason why it is designed so. Otherwise, it is a bug.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @UlyssesZh,

This is a known bug with Genymotion shell - we have already developed a fix and it will be implemented in the next stable release, Genymotion Desktop 3.3.0.

Best regards,
